server/src/main/java/org/candlepin/model/JobCurator.java

```diff
@@ -59,6 +61,7 @@ public JobStatus cancel(String jobId) {
         return result;
     }
 
+    @Transactional
```

Comment:
@mstead I dont know exact answer, but the reason we need to add Transactional is that new Hibernate doesn't allow running update HQL outside of a transaction. If you think about it it is more appropriate behavior anyway.